yb-voyager/src/migassessment/common.go

```diff
@@ -39,10 +39,13 @@ type SizingAssessmentReport struct {
 	MemoryPerInstance               float64
 	OptimalSelectConnectionsPerNode int64
 	OptimalInsertConnectionsPerNode int64
-	MigrationTimeTakenInMin         float64
-	ParallelVoyagerThreadsSharded   int64
-	ParallelVoyagerThreadsColocated int64
-	FailureReasoning                string
+	ImportTimeTakenInMin            float64
```

Comment:
nit: `EstimatedTimeInMinForImport` ? (even in the report)

Reply:
done. Changed it to ```EstimatedTimeInMinForImport``` .